binding/webgl_rendering_context.cc

```diff
@@ -2537,6 +2537,12 @@ napi_value WebGLRenderingContext::GetParameter(napi_env env,
   ENSURE_NAPI_OK_RETVAL(env, nstatus, nullptr);
 
   switch (name) {
+    case GL_MAX_VERTEX_TEXTURE_IMAGE_UNITS:
+    case GL_MAX_CUBE_MAP_TEXTURE_SIZE:
+    case GL_MAX_VERTEX_ATTRIBS:
+    case GL_MAX_VERTEX_UNIFORM_VECTORS:
+    case GL_MAX_VARYING_VECTORS:
+    case GL_MAX_FRAGMENT_UNIFORM_VECTORS:
```

Comment:
You'll need to export these on the GL object as well:

https://github.com/google/node-gles/blob/master/binding/webgl_rendering_context.cc#L585

Reply:
I haven't used N-API before this, so I'm not entirely sure what the NapiDefineIntProperty does, but aren't they defined here: https://github.com/google/node-gles/blob/master/binding/webgl_rendering_context.cc#L715 ?

Reply:
Oh nice - sorry I missed that this morning. Yes they are already exported.